i18n/errors/en.yml

```diff
@@ -1,10 +1,32 @@
 # Error definitions, usage Text.e.ERR999
+#
 # General format:
+# ERRORID
+#   display: optional hash of display options. See display_defaults below.
+#   text: |
+#      text of the message
+#
 # First Line: brief description of the error
 # Second line: blank
 # Third+ : detailed description, max 76 characters per line
+
 errors:
-  rendering_defaults: {header: true, footer: true, stack: true, log: true}
+  # These are the default display attributes for all messages.
+  # If you awnt to ovverride them for specific errors,
```

Comment:
I know it's only in a comment, but typos: `s/awnt to ovverride/want to override/`